ci_build_images/opensuse15.Dockerfile

```diff
@@ -0,0 +1,50 @@
+#
+# Buildbot worker for building MariaDB
+#
+# Provides a base OpenSUSE image with latest buildbot worker installed
+# and MariaDB build dependencies
+
+ARG base_image
+FROM "$base_image"
+LABEL maintainer="MariaDB Buildbot maintainers"
+
+# Install updates and required packages
+RUN zypper update -y && \
+    zypper install -y -t pattern devel_basis && \
+    zypper install -y git ccache subversion \
+    python-devel libffi-devel openssl-devel glibc-locale\
+    python-pip curl wget && \
+    # install MariaDB dependencies
+    #zypper mr -er repo-source
+    zypper -n si -d mariadb
+
+# Create buildbot user
+RUN useradd -ms /bin/bash buildbot && \
+    mkdir /buildbot && \
+    chown -R buildbot /buildbot && \
+    curl -o /buildbot/buildbot.tac https://raw.githubusercontent.com/MariaDB/mariadb.org-tools/master/buildbot.mariadb.org/dockerfiles/buildbot.tac
+WORKDIR /buildbot
+
+# upgrade pip and install buildbot
+RUN pip install -U pip virtualenv && \
+    pip install --upgrade setuptools && \
+    pip install buildbot-worker && \
+    pip --no-cache-dir install 'twisted[tls]'
+
+# Test runs produce a great quantity of dead grandchild processes.  In a
+# non-docker environment, these are automatically reaped by init (process 1),
+# so we need to simulate that here.  See https://github.com/Yelp/dumb-init
+RUN curl -Lo /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 && \
+    chmod +x /usr/local/bin/dumb-init
```

Comment:
Ideally this needs to be installed via the package manager

Reply:
Kept as is, because other Dockerfiles also install it this way - https://github.com/MariaDB/buildbot/blob/main/ci_build_images/centos7.Dockerfile#L55. Also, upgraded it to `1.2.5`.